### PR TITLE
230: upgrade ruby to 2.7.6

### PR DESCRIPTION
**Pull request by mikevallano**
## Related Issues & PRs
Closes #230

## Problems Solved
* Upgrade ruby to be able to upgrade heroku stack

## Screenshots
ruby-upgrade-server-running

## Things Learned
* If your circle config isn't valid, circle lets you edit it directly and does the linting to show you that it's valid, then you can commit in the interface and rerun the job 😎

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -42,16 +43,14 @@ group :development, :test do
   gem 'capybara-email'
   gem 'guard-rspec', require: false
   gem 'terminal-notifier-guard'
-  gem 'spring-commands-rspec'
-  gem 'spring'
```
Comment (mikevallano) [inline Gemfile:46]: These were causing errors in trying to load the rails console. I believe they're removed in a later version of rails anyway.

File: Gemfile [hunk 2]
```diff
@@ -42,16 +43,14 @@ group :development, :test do
...
 end
 
 group :test do
     gem 'shoulda-matchers'
     gem 'capybara'
     gem 'database_cleaner'
     gem 'launchy'
-    gem 'selenium-webdriver'
+    gem 'webdrivers'
```
Comment (mikevallano) [inline Gemfile:53]: Newer gem to handle this

File: .circleci/config.yml [hunk 3]
```diff
@@ -2,10 +2,13 @@
 # See: https://circleci.com/docs/2.0/configuration-reference
 version: 2.1
 
+orbs:
+  browser-tools: circleci/browser-tools@1.4.0
```
Comment (mikevallano) [inline .circleci/config.yml:6]: This is a thing from circle that fixed the webdriver problem: https://circleci.com/developer/orbs/orb/circleci/browser-tools

File: Gemfile (hunk 1 above)
Comment (lortza) [reply Gemfile:46]: 👍 I also believe that is true

**Review by lortza — APPROVED** [review]
That monkeypatch stuff is kind of weird, but since you explained it, I get it.

Everything else looks right on. 👍

**Pull request merged**